test/remote/gateways/remote_payu_latam_test.rb

```diff
@@ -82,8 +118,68 @@ def test_successful_purchase_brazil
     assert response.test?
   end
 
-  def test_successful_purchase_sans_options
-    response = @gateway.purchase(@amount, @credit_card)
+  def test_successful_purchase_colombia
```

Comment:
I'm curious about these country specific ones where we're adding the `payment_country` param to `options`- in the standard `test_successful_purchase` test above we don't add it to the options hash and it seems to be ok (assuming cause it's defaulting to "AR" give the changes here) so wondering what we're checking for each specific country and adding the `payment_country` field? Seems like if we didn't add it, then it would pass as well?

Reply:
Unfortunately either the sandbox isn't accurate, or the PaymentCountry field is not truly required for a successful transaction and is more of a soft requirement from PayU, because even without adding the field remote purchase tests succeed.

Reply:
So this test is more checking that the (supposedly) required country-specific fields are added, rather than that the transaction succeeds.

Reply:
> Unfortunately either the sandbox isn't accurate, or the PaymentCountry field is not truly required

☹️ 

> So this test is more checking that the (supposedly) required country-specific fields are added

I gotcha. So, if that's the case wondering if it would be better for us to do the [`check_request`](https://github.com/activemerchant/active_merchant/blob/d27c34e9e0e714afbf03d5eacf2a8cb20dc5b3da/test/unit/gateways/stripe_test.rb#L283) method of ensuring we are adding that field to the request if it exists in the options has AND that it overrides what is in the billing address. Given the value doesn't seem to matter against the sandbox, we can probably 🔪 each specific country and just ensure that we do send that field/it doesn't get accidentally removed. Basically, I feel dubious that changing out the country even gives us anymore confidence since it doesn't seem to be validated in sandbox?

Reply:
@davidsantoso I `check_request` for them in unit tests when they are options but I can add the check for overriding the billing address. It's possible they could add actual checking to their sandbox, so I'm not sure the remote tests should be removed though.

Reply:
Regardless of how the sandbox behaves, we should have cases that at a minimum exercise each country option.

Reply:
> It's possible they could add actual checking to their sandbox, so I'm not sure the remote tests should be removed though.

That's fine. However I'm still not convinced that it gives us any more confidence.